packages/cli/src/commands/certs/ls.ts

```diff
@@ -58,7 +58,7 @@ async function ls(
     output.log(formatCertsTable(certs));
   }
 
-  if (pagination && pagination.count === 20) {
+  if (pagination && pagination.count === paginationOptions[1]) {
```

Comment:
Is this a typo?

Reply:
I haven't updated the 4 other functions yet, because I wanted to make sure the way I did `alias` was ok. There's a few updates if `alias` works 👍🏾